spec/features/welcome_page/new_user_registration_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
```

Comment:
this spec is a bit confusing at first glance...it looks like we might be reiterating tests from other spec files?

what are the comments left in here for? will we need them in the future?